news/4358.bugfix

```diff
@@ -0,0 +1,2 @@
+Not a feature because the ``hg+file`` scheme was already functioning implicitly but inconsistently documented.
```

Comment:
This file is User Facing. I think a `.doc` file saying the following would suffice:

```
Correct inconsistency related to the `hg+file` scheme.
```